docs/nddata/index.rst

```diff
@@ -156,6 +157,57 @@ Code that uses the arithemtic methods that used to be included in
 instead subclass `~astropy.nddata.NDDataArray`; that class is equivalent to
 the original `~astropy.nddata.NDData` class.
 
+.. _nddata_transition2:
+
+Transition to astropy x.xx
+==========================
```

Comment:
I think this should be in the changelog rather than a separate section in the sphinx documentation. All of the detailed changes show up in the docstrings and examples, and